_data/chains/eip155-10522.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "xMltec",
+  "chain": "xMLTEC",
+  "icon": "xmltec",
+  "rpc": [
+    "http://116.204.64.12:8545"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "xMLTEC",
+    "symbol": "xMLTEC",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.mylandtec.com",
+  "shortName": "XMT",
+  "chainId": 10522,
+  "networkId": 10522,
+  "slip44": 700,
+  "explorers": [{
+    "name": "xmltecscan",
+    "url": "https://e.mylandtec.com",
```

Comment:
I cannot reach this site

Reply:
Thanks for your reminding. I will improve it in the near future.
Thank you.^..^

Reply:
> I cannot reach this site

I need help plz

Reply:
I need help